### DISCUSSION TITLE
Disable Version Control Integration

### DISCUSSION THREAD

**Discussion by internalbuilds** [WeblateOrg/weblate#12008] [Q&A]
I want to use Weblate where  the ONLY use is for translators to view/translate strings.
Adding strings that need to be translated and Pulling translated strings from Weblate will be done via Weblate's Components/Units APIs.

To achieve the above, I want to "disable" any processing that Weblate does with version control. Looking for I go about do this.

**Comment by nijel** [comment]
You can remove all but local backends from https://docs.weblate.org/en/latest/admin/config.html#vcs-backends

**Reply by internalbuilds** [reply]
Thanks for the quick response @nijel.

I tired what you said. I am running on the Docker image. I added VCS_BACKENDS = ("weblate.vcs.git.LocalRepository",) to my existing "settings-override.py" file and re-launched, but I did not notice a change. 

Is there a way in the UI (or DB) that I can verify that the only VCS backends are local repos ?

**Reply by nijel** [reply]
When you configure a new component, it should not offer you anything else than local. But I think the initial forms will stay there, the code is not really prepared for this option.
